service_mesh/service_mesh_install/prepare-to-deploy-applications-ossm.adoc

```diff
@@ -0,0 +1,24 @@
+[id="deploying-applications-ossm"]
+= Deploying applications on {ProductName}
+include::modules/ossm-document-attributes.adoc[]
+:context: deploying-applications-ossm
+
+toc::[]
+
+When you deploy an application into the {ProductShortName}, there are several differences between the behavior of applications in the upstream community version of Istio and the behavior of applications within a {ProductName} installation.
+
```

Comment:
I might add a prereqs section and xref to at least one of the installation assemblies.

Reply:
I added Prerequisites with xrefs to the Istio vs OSSM comparison and the installation assembly.